docs/index.md

```diff
@@ -48,11 +48,26 @@ book_library:
 
 * As a library:
 
+  For multiple files
+
     ```python
     from yamlfix import fix_files
 
     fix_files(['file.py'])
     ```
+  where `'[file.py]`can be replaced by using the following expression:
+    ```python
+    file_a = TextIOWrapper(open('fileA.yml', 'r+b'))
```

Comment:
You should not need to do all this to open two files, `fix_files(['fileA.yml', 'fileB.yml'])` should work, I'll upload a fix for the bug as soon as I can

Reply:
But I agree that we can change the line 56 to `fix_files(['fileA.yml', 'fileB.yml'])`